### PR TITLE
Timer in results now human readable

### PR DESCRIPTION
**Pull request by wavell**
## Description
(name of issue/change)

## Issues:
Refs: #NNN

## How has this been tested:
 - [ ] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [ ] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [ ] Kind cluster
 - [ ] Have not tested

## Types of changes:
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation update

## Checklist:
**Documentation**
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [ ] Tasks in issue are checked off

### CODE REVIEW

**Review by agentpoyo — APPROVED** [review]
LGTM

File: src/tasks/utils/points.cr [hunk 1]
```diff
@@ -436,9 +436,11 @@ module CNFManager
       cmd = "#{Process.executable_path} #{ARGV.join(" ")}"
       Log.info {"cmd: #{cmd}"}
       end_time = Time.utc
-      task_runtime = (end_time - start_time).milliseconds
+      # task_runtime = (end_time - start_time).milliseconds
+      task_runtime = (end_time - start_time)
+      task_runtime_human = "#{task_runtime.hours} hours, #{task_runtime.minutes} minutes, and #{task_runtime.seconds} seconds"
```
Comment (martin-mat) [inline src/tasks/utils/points.cr:441]: I'd suggest to make is simpler. We are putting it to a yaml for possible engine postprocessing. %H:%M:%S is a standardized format that everyone understands and easily parsable. And task_runtime (Time::Span) as it is now will already provide this format when interpreted as a string. So From this place down, task_ruintime can be kept.

File: src/tasks/utils/points.cr [hunk 2]
```diff
@@ -475,7 +477,7 @@ module CNFManager
                    exit_code: results["exit_code"],
                    items: result_items}, f)
       end
-      Log.info { "upsert_task: task: #{task} has status: #{status} and is awarded: #{points} points. Runtime: #{task_runtime} seconds" }
+      Log.info { "upsert_task: task: #{task} has status: #{status} and is awarded: #{points} points. Runtime: #{task_runtime_human} seconds" }
```
Comment (martin-mat) [inline src/tasks/utils/points.cr:480]: remove "seconds" at the end of the string.

**Review by martin-mat — CHANGES_REQUESTED** [review]

**Comment by martin-mat** [comment]
Remove this PR, a duplicate PR was merged before, such mess was created by opening a duplicate PR (and ignoring review comments in this one)

**Pull request closed without merging**